src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -514,15 +514,18 @@ event::ptr primitive_inst::realloc_if_needed() {
     // Update param if fake_alignment is available
     auto updated_params = get_fake_aligned_params_if_possible(*_impl_params);
 
-    const auto& actual_layout = updated_params.get_output_layout();
-    OPENVINO_ASSERT(actual_layout.is_static(), "[GPU] Can't realloc mem for dynamic layout");
+    const auto& actual_layouts = updated_params.output_layouts;
+    OPENVINO_ASSERT(actual_layouts[0].is_static(), "[GPU] Can't realloc mem for dynamic layout");
 
     // input_layout node is supposed to always use external memory in dynamic case
     if (_node->is_type<input_layout>())
         return ev;
 
     auto& sp = *get_network().get_shape_predictor();
-    auto dt_size = ov::element::Type(actual_layout.data_type).bitwidth();
+    std::vector<size_t> dt_sizes;
+    for (size_t i = 0; i < actual_layouts.size(); ++i) {
+        dt_sizes.push_back(ov::element::Type(actual_layouts[i].data_type).bitwidth());
```

Comment:
Looks like we use byte size everywhere, so I suppose we can replace `bitwidth()` here with `size()`

Reply:
Fixed